README.md

```diff
@@ -35,15 +29,20 @@ Configuration example:
 apiVersion: th2.exactpro.com/v1
 kind: Th2GenericBox
 spec:
-  image-name: your.image.repo:42/th2-conn-template
+  image-name: your.image.repo:42/th2-conn-amqp
   image-version: 0.0.1
   type: th2-conn
   custom-config:
       sessionAlias: session-alias
       drainIntervalMills: 1000
       rootEventName: ConnTemplate
       parameters:
-          # your custom parameters here if you need them
+          # initialContextFactory:
+          # factorylookup:
+          # username:
+          # password:
+          # sendQueue:
+          # receiveQueue:
```

Comment:
Different types of data can be transferred via MQ by different queues and routing keys.
th2 can route streams of data by session_alias. the feature to define several session aliases for several routing keys and queues will be useful